### PR TITLE
Created a Paragraph Summerizer #720

### PR DESCRIPTION
**Pull request by sushilmagare10**
**Please describe the changes this PR makes and why it should be merged:**

Added the Paragraph Summerizer
summerizer

**Status**

- [] Code changes have been tested against prettier, or there are no code changes
- [ ] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**

- [ ] This PR changes the codebase
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
  - [ ] This PR changes the internal workings with no modifications to the external API (bug fixes, performance improvements)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

this pr solves issus #720

### CODE REVIEW

File: packages/kitchen-sink/src/examples/paragraph-summerizer.jsx [hunk 1]
```diff
@@ -0,0 +1,256 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+const paragraphSummerizer = () => {
```
Comment (tobySolutions) [inline packages/kitchen-sink/src/examples/paragraph-summerizer.jsx:4]: Hey there @sushilmagare10, I don't see your usage Million.js anywhere 🤔.

File: packages/kitchen-sink/src/examples/paragraph-summerizer.jsx [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setSubmitting(true);
+
+    const requestOptions = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: 'Bearer ' + import.meta.env.VITE_OPENAI_API,
```
Comment (tobySolutions) [inline packages/kitchen-sink/src/examples/paragraph-summerizer.jsx:18]: I don't think this is working. Cos, it seems like you need an API key that you are not providing in production which could cause the deployed version of this to not work. I'll suggest using another alternative or any provider.


https://github.com/aidenybai/million/assets/96334363/52eb82c1-bb81-4ce9-a5c8-00b4b1b40da2

**Comment by sushilmagare10** [comment]
I see . I try to do it other way and aslo use the million Block... @tobySolutions

**Comment by tobySolutions** [comment]
Awesome, I'm reviewing it right now.

**Comment by tobySolutions** [comment]
> I see . I try to do it other way and aslo use the million Block... @tobySolutions

Hey there @sushilmagare10, any updates on this?

**Comment by sushilmagare10** [comment]
This uses the Block and added API key in it @tobySolutions

**Comment by tobySolutions** [comment]
> This uses the Block and added API key in it @tobySolutions

Hmm, can you recheck, please? It doesn't seem like you've made any updates

**Pull request closed without merging**